src/components/dashboard.jsx

```diff
@@ -18,6 +18,18 @@ const SAMPLE_REPOS = [
   {repoOwner: 'jquery', repoName: 'jquery'}
 ];
 
+try {
```

Comment:
What causes this code to require a `try/catch`?

Reply:
When not in production mode, REPOSITORIES will be undefined, leading to an exception.

Maybe I should change that logic a bit in webpack.config.js?

Reply:
You have all your code wrapped in an `if` statement, though, so none of the code will be run if `REPOSITORIES` is `undefined`, right?

Reply:
Check for the type instead `typeof REPOSITORIES !== 'undefined'`